src/test/kotlin/com/zepben/evolve/streaming/get/NetworkConsumerClientTest.kt

```diff
@@ -1018,8 +1018,12 @@ internal class NetworkConsumerClientTest {
                 isSeriesCompensator = { seriesCompensator = it.toPb() },
                 isGround = { ground = it.toPb() },
                 isGroundDisconnector = { groundDisconnector = it.toPb() },
+                isGroundingImpedance = { groundingImpedance = it.toPb() },
```

Comment:
todo: Should be reordered for revised order in the underlying class.

Reply:
Reordered